webapp/src/components/modals/create_issue/create_issue_form.tsx

```diff
@@ -136,7 +136,8 @@ export default class CreateIssueForm extends React.PureComponent<Props, State> {
         });
 
         let fields = {
-            ...this.state.fields,
+            summary: this.state.fields.summary,
+            description: this.state.fields.description,
```

Comment:
Changes LGTM, though there is one other case I want to test @dipak-demansol. We have solved this for switching projects, but not for switching issue types. A similar change could be done on line 164 to fix this.

@maisnamrajusingh I would go ahead and apply the same change there to avoid a crash in that scenario too. This would happen if a given issue type schema has a field that another issue type in the same project does not have. Even though that is probably uncommon, I'm thinking we should be ssafe there.

The drawback is that the user would lose some progress on filling th eissue out when they change issue types, but this way is much more correct and solves other potential issues as well.

Reply:
@dipak-demansol I don't think it's worth the trouble to set up two issue schemas in the same Jira project that have the characteristics I brought up above. No need to try to have a reproducible case for that scenario.

Reply:
Hi! Can we solve this conversation? I think there nothing else to be fixed here right? Let me know if I can help in any way here. @mickmister

Reply:
> We have solved this for switching projects, but not for switching issue types. A similar change could be done on line 164 to fix this.

@javaguirre I think this request is still pending a change here

Reply:
are you working on this @maisnamrajusingh ?

Reply:
@javaguirre This PR was submitted by a previous contractor, so it's sort of in a frozen state to be used as a starting point for someone else to pick up the task. I've marked the ticket https://github.com/mattermost/mattermost-plugin-jira/issues/819 as HW and linked to this PR for reference. I'll close this since the source branch won't be used. Thanks for circling back to this PR :+1: